index.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Matthew Tapley Portfolio</title>
+    <link rel="stylesheet" type="text/css" href="styles.css">
+</head>
+<a class="anchor" id="home"></a>
+<body>
+    <nav>
+        <ul class="navLinks">
+            <a href="#home">HOME</a>
+            <a href="#about">ABOUT</a>
+            <a href="#portfolio">PORTFOLIO</a>
+        </ul>
+        <div class="hamburgerMenu">
+            <div></div>
+            <div></div>
+            <div></div>
+        </div>
+    </nav>
+
+    <div class="homePage">
+        <div class="homeText">
+            <h1>MATTHEW TAPLEY.csm</h1>
+            <sub>TRAINEE FULL-STACK DEVELOPER</sub>
+        </div>
+        <div class="contactIcons">
+            <a href="https://github.com/mattap86">
+                <img class="gitHub" src="Images/github-57px.png">
+            </a>
+            <a href="#https://twitter.com/mattap86#">
+                <img class="twitter" src="Images/twitter-64px.png">
+            </a>
+            <!--<a href="#composeNewEmail#">-->
+                <img class="email" src="Images/envelope-64px.png">
+            </a>
+            <a href="https://www.linkedin.com/in/matthew-tapley-5b5a0ba0/">
+                <img class="linkedIn" src="Images/linkedin-64px.png">
+            </a>
+        </div>
+    </div>
+    <a class="anchor" id="about"></a>
+    <div class="aboutMe">
+        <div class="aboutText">
+            <h1>ABOUT ME</h1>
+            <sub>Some Info About Me. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec at magna mi. Nullam mattis leo tellus, a ultrices purus semper vel. Praesent ornare sapien sit amet auctor dapibus. Vivamus et lobortis magna, vel volutpat erat. Fusce eros neque, feugiat ac ipsum in, luctus faucibus neque. Donec rhoncus ex vel lectus congue auctor. Sed porttitor neque finibus sem gravida lobortis.</sub>
+        </div>
+        <div class="aboutMeBoxes">
+            <button class="interests" type="checkbox">INTERESTS</button>
+            <button class="qualifications">QUALIFICATIONS & ACCOLADES</button>
+            <button class="something">SOMETHING ELSE ABOUT ME</button>
+            <a class="anchor" id="portfolio"></a>
+        </div>
+    </div>
+    <div class="portfolioText">
+        <h1>PORTFOLIO</h1>
+        <sub>A Look At My Projects</sub>
+        <div class="spacer"></div>
+    </div>
+    <div class="portfolio">
+        <div class="projectBlocks">
+            <!--All commented links will be images rather than text (lines 67 through 107 where relevant-->
+            <div class="projectOne">
+                <div class="project projectBox">
+                    <a href="file:///Users/academy/Documents/Projects/PilotShop/pilotshop.html">
```

Comment:
I can't see what that is. so for someone who doesn't have the same file on their computer, it would be wise to use "alt" attribute in the img tag to help describe what it is if the page doesn't load.

Reply:
have mentioned that in the initial comment at top of pull request. was left in to show images will be inserted in the boxes